.travis.yml

```diff
@@ -15,9 +15,9 @@ services:
 
 language: scala
 scala:
-  - 2.11.12
+  - 2.12.13
 jdk:
-  - oraclejdk8  # sbt +compile fails on jdk11 (xenial's default)
+  - openjdk8
```

Comment:
should we do the java 11 upgrade before the release or 0.18? what do you think? @voganrc @jsleight

Reply:
imo we should do the 0.18 release before java 11 for a couple main reasons:
1. It gets the spark 3 release out sooner, which many people have been requesting
2. It gives people an intermediate release which they can use on spark 3 and java 8.

Reply:
👍